### PR TITLE
Added BuildRecord REST with RSQL

### PR DESCRIPTION
**Pull request by vibe13**

### CODE REVIEW

**Comment by vibe13** [comment]
test this please

File: datastore/src/main/java/org/jboss/pnc/datastore/predicates/rsql/RSQLNodeTravellerPredicate.java [hunk 1]
```diff
@@ -1,78 +1,91 @@
...
+    }
 
-      PathBuilder<Entity> pathBuilder = new PathBuilder<>(selectingClass, selectingClass.getSimpleName().toLowerCase());
+    @Override
+    public BooleanExpression get() {
 
-      RSQLNodeTraveller<BooleanExpression> visitor = new RSQLNodeTraveller<BooleanExpression>() {
+        // Using lower-cases string variables makes Entities with camelCase names unusable (i.e. BuildRecord)
+        PathBuilder<Entity> pathBuilder = new PathBuilder<>(selectingClass, Introspector.decapitalize(selectingClass
+                .getSimpleName()));
```
Comment (vibe13) [inline datastore/src/main/java/org/jboss/pnc/datastore/predicates/rsql/RSQLNodeTravellerPredicate.java]: @slaskawi I had to change into Introspector.decapitalize(selectingClass.getSimpleName())), as the Entities with camelCase name could not be resolved. In this way I will put "buildRecord" and not "buildrecord", which does not get resolved

Comment (slaskawi) [inline datastore/src/main/java/org/jboss/pnc/datastore/predicates/rsql/RSQLNodeTravellerPredicate.java]: +1, thanks!

**Comment by vibe13** [comment]
@slaskawi  ok merging. thanks!

**Pull request merged**